### DISCUSSION TITLE
cn/2024/09/29/t/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#181] [Announcements]
# cn/2024/09/29/t/

https://hongtaoh.com/cn/2024/09/29/t/

**Comment by hongtaoh** [comment]
@earfanfan 统计大佬能不能给这篇顺带前两篇提一些建议。

**Reply by earfanfan** [reply]
阿，我好菜的说，不过我可以召唤一下真正的大佬。@XiangyunHuang，湘云，你快来看看呀。

**Reply by hongtaoh** [reply]
@XiangyunHuang 确实是大牛。看到他同时在写好几本书，看了他的内容我感觉我还有好多要学的呀。太多不会了。我现在感觉是很多方法如果只会用不知道底层原理 (至少要知道自己哪里不知道)，最后就什么都记不住，只知道几行代码而已。统计和机器学习教育一个很大的问题是只讲是什么、怎么做，但是不讲为什么，因为讲起来很费力，大多数人就一带而过了，最后愿意刨根问底的人很少。

**Reply by XiangyunHuang** [reply]
我觉得写得很好呀。你想知道为什么的部分，我感觉需要看统计发展史一类的书。还有那些统计学家的人物传记，里面会提到他们为什么那样想、那样做。

**Reply by XiangyunHuang** [reply]
我那几本笔记刨根问底的很少，只做了一些分类整理。真的大牛是益辉、肖楠等人，他们对基本概念理解得深。

**Reply by XiangyunHuang** [reply]
除了历史传记，还有一个地方，就是这些概念提出的原始论文，不过，因为这些论文很早，英文用词和含义与100 年后的今天会有所不同，读起来会更麻烦些，有的学者特意去挖这些论文搞研究。P.S. 我曾尝试过关于二项分布的覆盖概率与成功概率的关系的了解。

**Reply by hongtaoh** [reply]
@XiangyunHuang 对于非专业科研者，而只是用的话，感觉连基本概念也许都不需要弄清楚，会用 R 处理和分析数据就可以了，只不过我很难接受去用一些我完全不懂的东西。这次整理一下最基本的概念，我内心多了一些安全感，我知道了我的知识边界，比如对于 t 分布的密度函数推导，我现在无法了解，那我去用就好了，不一定非要自己推出来。

你提到的那本教材我刚才看了。说实话，我根本不可能有心力去把那本教材一字一句看下来。@earfanfan，我一直提倡要[把教材当成字典](https://hongtaoh.com/cn/2022/11/25/textbook-dict/)。不需要去把字典一字一句看完，你只需要用的时候去翻相关的章节就可以了。我私以为中国人学东西最可怕的一点就是想「系统性」学一下。这么想的话恐怕永远都无法开始。「系统性学一下」的结果就是 99% 的时间花在了 1% 的知识点上。

**Reply by XiangyunHuang** [reply]
我觉得你是对的，只需要把一些基础的、常用的抠明白就行，其他的可以按需学习。

**Comment by fenguoerbian** [comment]
t分布需要分子上的正态分布随机变量和分母上的卡方随机变量是独立的，这里漏掉了这个陈述。对于来自于正态分布的iid样本而言，样本均值与样本方差的分布恰好是独立的，虽然它们都是由x1到xn这n个样本计算得到的结果。

**Reply by hongtaoh** [reply]
超级感谢专业人士的指点！！很感激。你提到的这些我要先学习消化一下，看来学无止境。

**Comment by yuzhao66** [comment]
这学期的计量经济学正好学到这里了，谢谢师兄贴出来这些图，看评论区也收获到了很多。
